### ISSUE TITLE
Welcome to the A11Y Project

### ISSUE THREAD

**Issue by davatron5000** [a11yproject/a11yproject.com#1]
Thanks so much for your interest in helping make the web more accessible. Welcome and I'm so glad you're here. This site is as much yours as it is mine. 

So the pitch:
I've been making web sites for over a decade now but web accessibility (A11Y) still seems to be the area where it's most difficult to grasp best practices. Furthermore, when you search for information, it's buried in long posts. I want to make A11Y easier to digest so that designers and developers can make more easily these considerations and make the web a better place.

Awhile back I created [a list of ~40 simple posts on A11Y that I would love to see written](https://gist.github.com/a43fa81793491431c13d). It's by no means the end-all-be-all, but it should be straightforward on the type of posts I'm after: small, topical and friendly. 

And because this is all on Github, information can always be the latest and have the input of the whole web. Here's a list of people who said they were interested or have mentioned it in the past:

@wilto
@ianhamilton
@bryanstedman
@sturobson
@susanjrobertson
@mattmcmanus
@grayghostvisuals
@joshualong
@jasonrhodes

The site isn't ready for launch yet by any means, so keep it a bit of a secret right now. I'll start logging issues that I have with the site at the moment and assign some to myself and start clearing them out.

If you are jekyll and github savvy and you'd like to become an admin, just post an :octocat: 

Thanks again for helping,
Dave
